README.md

```diff
@@ -25,24 +25,41 @@ The commands can be enabled or disabled via two boolean inputs:
 name: Configlet
 
 on:
-  push:
   pull_request:
+    branches:
+      - main
```

Comment:
Is the branch restriction wanted here? If I open a PR against another PR branch, I'd still want it to run configlet, I think.

Reply:
🤔 I think you're right. Let me update